proto/viam/app/v1/app.proto

```diff
@@ -723,6 +725,16 @@ message ListAuthorizationsResponse {
   repeated Authorizations authorizations = 1;
 }
 
+message GetAuthorizationsForUserInOrgRequest {
+  string identity_id = 1;
+  string organization_id = 2;
+}
+
+message GetAuthorizationsForUserInOrgResponse {
+  string identity_id = 1;
```

Comment:
You can remove this. We should already have the identity_id from when send the GetAuthorizationsForUserInOrgRequest